Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Here the version of the registry is specified this storage branch uses.
+# It should always be a specific version to make sure builds are reproducible.
+ARG PACKAGE_REGISTRY=57d417a84fa038cfb50a0cd661372c194199b3fd
```

Comment:
@jsoriano Registry version is fixed. Here it is a commit hash but i expect it will become a released version soon.